diff --git a/include/RE/V/VirtualMachine.h b/include/RE/V/VirtualMachine.h
index 6b715ee1..f65f78f7 100644
--- a/include/RE/V/VirtualMachine.h
+++ b/include/RE/V/VirtualMachine.h
@@ -218,7 +218,7 @@ namespace RE
 				stl::enumeration<FreezeState, std::uint32_t>               freezeState;                 // 939C
 				mutable BSSpinLock                                         attachedScriptsLock;         // 93A0
 				BSTHashMap<VMHandle, BSTSmallSharedArray<AttachedScript>>  attachedScripts;             // 93A8
-				std::uint32_t                                              unk93D8;                     // 93D8
+				std::uint32_t                                              nextObjectToClean;                     // 93D8
 				std::uint32_t                                              unk93DC;                     // 93DC
 				BSTArray<BSTSmartPointer<Object>>                          objectsAwaitingCleanup;      // 93E0
 				mutable BSSpinLock                                         arraysLock;                  // 93F8
diff --git a/include/REL/Module.h b/include/REL/Module.h
index 54a484c7..e2ce4dcc 100644
--- a/include/REL/Module.h
+++ b/include/REL/Module.h
@@ -294,6 +294,11 @@ namespace REL
 					}
 				}
 			}
+			else {
+				_filename.resize(_filename.size() - 1);
+				moduleHandle = REX::W32::GetModuleHandleW(_filename.c_str());
+			}
+
 			_filePath = _filename;
 			if (!moduleHandle) {
 				stl::report_and_fail(
